wordpress-mobile/WordPress-iOS
# Issue/5665 photo goes missing

Fixes #5665 

This bug shows up because of  interesting symlink returned on URL's coming ```[NSFileManager contentsOfDirectoryAtURL]``` more details here: http://stackoverflow.com/questions/14992834/what-does-the-private-prefix-on-an-ios-file-path-indicate

To test:
 - Check the steps to reproduce on the issue #5665 and make sure that images are not deleted.

Needs review: @aerych
